guide/inflation.rst

```diff
@@ -268,15 +283,31 @@ save the entire series of inflation files to explore how inflation evolves throu
 As part of the workflow, you have to take the output of one assimilation cycle and rename
 it to be the input for the next assimilation cycle. That is the time to make a copy
 that has a unique name - usually with some sort of date or timestamp. This also makes
-it possible to restart an experiment.
-
-.. important::
-		Inflation is only applied to the variables that are **updated** by DART.
-		If you have ``NO_COPY_BACK`` variables in your state, then inflation
-		will not impact them. They are often used to compute the forward
+it possible to restart an experiment. For many of the supported models, the scripting that
+does the renaming and copying of the inflation files may already be provided.
+
+.. Important::
+		Inflation is only applied to the variables that are **updated** by DART
+                by specifying ``UPDATE`` within the ``&model_nml`` section of ``input.nml``.
+		Alternatively, if the variables are not updated (``NO_COPY_BACK``) then inflation
+		will not impact them. ``NO_COPY_BACK`` variables are often used to compute the forward
 		operators, however, they don't take part in the update and as such
 		they will not be inflated.
 
+                Assigning a variable as ``NO_COPY_BACK`` is the **only way** to prevent inflation
+                from being applied to a variable when inflation is turned on.  If a variable is
+                not updated by observations because either 1) observation gaps exist in space or time 
+                2)the state is outside the spatial localization distance, or  3) variable localization 
+                was applied through the ``obs_impact_tool``, --the inflation update for that part of the
```

Comment:
inflation damping happens whether inflation was updated with observations or not. I am not a fan of the word 'revert', I'll edit the text.

Reply:
I'm on it. I realize that I needed more clarity so I added a note detailing the order of inflation operations in DART. Will push shortly